apps/web/app/future/auth/verify/page.tsx

```diff
@@ -28,9 +26,15 @@ export const generateMetadata = async ({ params, searchParams }: _PageProps) =>
     );
   }
 
-  const data = await StripeRepository.getCheckoutSession({
-    stripeCustomerId,
-    checkoutSessionId: sessionId,
+  const data = await fetch("/api/v2/stripe/get-checkout-session", {
```

Comment:
We made the decision to not have the web app hit API v2. We should use existing tRPC calls for this.

Reply:
Got it. Just made the change way simpler